crates/router/src/types/api/admin.rs

```diff
@@ -192,6 +194,8 @@ impl ForeignTryFrom<(domain::MerchantAccount, BusinessProfileCreate)>
                 .or(Some(true)),
             collect_shipping_details_from_wallet_connector: request
                 .collect_shipping_details_from_wallet_connector,
+            collect_billing_details_from_wallet_connector: request
+                .collect_billing_details_from_wallet_connector,
```

Comment:
we can provide default values for these fields from the application instead of relying on the database